docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,302 @@
+- `c`, denoting the number of connections from the load runner process to the target node, and
+- `r`, indicating the rate or frequency of transactions issued per second. Each connection
+  dispatches `r` transactions per second. 
+
+For more details on the methodology to identify the saturation point, see
+[here](CometBFT-QA-34.md#saturation-point).
+
+The figure below shows the values obtained for v1 and v0.38 (the baseline). It's important to note
+that configurations that have the same amount of total transaction load are regarded as equivalent.
+For example, `c=1,r=400` and `c=2,r=200` are plotted on the same x-axis value corresponding to their
+total rate of 400 tx/s, which corresponds to configuration with `c=1`.
+
+![saturation-plot](imgs/v1/saturation/saturation_v1_v038.png) 
+
+In the figure, we observe that up to a rate of 400 tx/s, the obtained values closely match or are
+equal to the expected number of processed transactions, which is 35600 txs. However, beyond this
+point, the system becomes overloaded and cannot process all incoming transactions, resulting in
+dropped transactions. This state indicates that the system is saturated. The expected number of
+processed transactions is calculated as `c * r * 89 s = 35600 txs`. It's worth noting that we
+utilize 89 out of 90 seconds of the experiment duration, as the final transaction batch coincides
+with the end of the experiment and is thus not sent.
+
+The complete results from which the figure was generated can be found in the file
+[`v1_report_tabbed.txt`](imgs/v1/200nodes/metrics/v1_report_tabbed.txt). The following table
+summarizes the values plotted in the figure. We can see the saturation point in the diagonal defined
+by `c=1,r=400` and `c=2,r=200`.
+
+| r    | c=1       | c=2       | c=4   |
+| ---: | --------: | --------: | ----: |
+| 200  | 17800     | **34600** | 50464 |
+| 400  | **31200** | 54706     | 49463 |
+| 800  | 51146     | 51917     | 41376 |
+| 1600 | 50889     | 47732     | 45530 |
+
+For comparison, this is the table obtained on the baseline version, with the same saturation point.
+
+| r    | c=1       | c=2       | c=4   |
+| ---: | --------: | --------: | ----: |
+| 200  | 17800     | **33259** | 33259 |
+| 400  | **35600** | 41565     | 41384 |
+| 800  | 36831     | 38686     | 40816 |
+| 1600 | 40600     | 45034     | 39830 |
+
+In conclusion, we chose `c=1,r=400` as the transaction load that we will use in the rest of QA
+process. This is the same value used in the previous QA tests.
+
+### With latency emulation
+
+For comparing the performance of `v1` with and without latency emulation, we run a new set of
+experiments with a different configurations of transaction loads: we use only one connection, and a
+transaction rate ranging from 100 to 1000 tx/s, in intervals of 100.
+
+![v1_saturation](imgs/v1/saturation/saturation_v1_LE.png) 
+
+These are the actual values which we use to generate the figure:
+
+| r    | v1 without LE | v1 with LE   | 
```

Comment:
Hmm, _with_ latency emulation the system has _higher_ throughput. 

1. Is this because the system is forced to trade latency for throughput?
2. Also, we're using identical node configuration for the two runs (with/without)?

Nit: Related to my second point, it might help to recall the exact config params we're using, maybe give link to the config.toml that was used?

Reply:
> 2. Also, we're using identical node configuration for the two runs (with/without)?

Yes, it's the same default configuration on both cases. I'll clarify it in the text.

Reply:
> Hmm, with latency emulation the system has higher throughput.

I initially saw the same, so I run a couple of experiments more. Please check out the updated the image and text.